msgbox/msgbox.js

```diff
@@ -69,14 +69,21 @@ function messageBox({
           onclick: messageBox.listeners.closeIcon}),
         $element({id: `${id}-contents`, appendChild: tHTML(contents)}),
         $element({id: `${id}-buttons`, appendChild:
-          buttons.map((textContent, buttonIndex) => textContent &&
-            $element({
+          buttons.map((textContent, buttonIndex) => {
+            if (!textContent) {
+              return;
+            }
+            let onclick = messageBox.listeners.button;
+            if (typeof textContent === 'object') {
```

Comment:
Now it's a bad name for the parameter.
Something like this would be more consistent:

```js
          buttons.map((content, buttonIndex) => content && $element({
            tag: 'button',
            buttonIndex,
            textContent: content.textContent || content,
            onclick: content.onclick || messageBox.listeners.button,
          }))
```